### ISSUE TITLE
Timeout

### ISSUE THREAD

**Issue by retog** [rdf2h/browser#2]
If a request takes too long (such as when it's blocked by cors) the user should get a meaningful message. This could help: https://italonascimento.github.io/applying-a-timeout-to-your-promises/

**Comment by retog** [comment]
With Updated rdfgraphnode clocked request no longer results in promises taht never get rejected, so this in no longer needed to work around this problem.
